pyodide_build/pypabuild.py

```diff
@@ -34,8 +35,14 @@
 AVOIDED_REQUIREMENTS = [
     # mesonpy installs patchelf in linux platform but we don't want it.
     "patchelf",
+    "oldest-supported-numpy",
 ]
 
+AVOIDED_REQUIREMENTS: list[str] = [
+    "patchelf",
+    "oldest-supported-numpy",
+] + get_host_build_flag("AVOIDED_BUILD_REQUIREMENTS").split()
```

Comment:
Instead of appeding the value, let's make the default value of `AVOIDED_BUILD_REQUIREMENTS` to `patchelf` and `oldest-supported-numpy`. If one of the `patchelf` or `oldest-supported-numpy` causes trouble downstream, I want to let people remove it.

Reply:
Thanks Gyeongjae, addressed.